codalab/apps/newsletter/forms.py

```diff
@@ -0,0 +1,29 @@
+from django import forms
+
+from .models import NewsletterSubscription
+
+
+class NewsletterSubscriptionSignUpForm(forms.ModelForm):
+    class Meta:
+        model = NewsletterSubscription
+        fields = ['email']
+
+        def clean_email(self):
+            email = self.cleaned_data.get()
+
+            return email
+
+
+class NewsletterSubscriptionUnsubscribeForm(forms.ModelForm):
```

Comment:
We should probably use `forms.Form` instead of a `ModelForm`, then we don't need to override anything about unique values.

Reply:
Updated in latest push